tests/base.py

```diff
@@ -0,0 +1,11 @@
+import maya.cmds as mc
```

Comment:
To keep the `cmds` vs `mc` consistent it would be great to use `cmds` here, too. Otherwise it's the one exception :)

Reply:
Addressed this in the most recent commit.  Marking as resolved.